### DISCUSSION TITLE
Plans for builtin:appsec.notification-integration support

### DISCUSSION THREAD

**Discussion by reinhard-brandstaedter** [Dynatrace/dynatrace-configuration-as-code#1618] [Ideas]
I was surprised to not find support for the security notification profiles.
Are there any plans to add this in the near future? I've neatly created ownerships and standardized notification profiles for teams and wanted to do the same for security related issues, but currently thats not possible.

**Comment by Laubi** [comment]
This is a settings configuration and is supported by utilizing the `settings` type in Monaco. 
Please note that we will no longer support any questions here on GitHub. For any questions, please reach out to our official Dynatrace support: https://support.dynatrace.com/
